packages/office-addin-debugging/src/start.ts

```diff
@@ -218,12 +219,11 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
             throw new Error("Manifest does not contain the id for the Office Add-in.");
         }
     
-        // DISABLE FOR NOW
         // enable loopback for Edge
-        // if (isWindowsPlatform) {
-        //     const name = isDesktopAppType ? "EdgeWebView" : "EdgeWebBrowser";
-        //     await devSettings.ensureLoopbackIsEnabled(name);
-        // }
+        if (isWindowsPlatform && parseInt(os.release()) === 10) {
```

Comment:
It would be a bit more readable if add `const isWindows10: boolean = isWindowsPlatform && parseInt(...);` on line 199 just after the declaration of isWindowsPlatform, so that here you can just use `if (isWindows10)`

Reply:
@TCourtneyOwen did you make a change here?